### ISSUE TITLE
Ask polkit whether a user has admin rights

### ISSUE THREAD

**Issue by AceBlade258** [cockpit-project/cockpit#4349]
(ticket title suggested by helpful person in IRC, I only kinda know what I'm doing)
OS is CentOS 7, cockpit version 0.96, joined to a FreeIPA domain

/etc/polkit-1/rules.d/40-freeipa.rules contains

```
polkit.addAdminRule(function(action, subject) {
    return ["unix-group:admins", "unix-group:wheel"];
});
```

I am unable to make most changes; I can turn on and off the individual adapters in my one bond, and could theoretically create new pools in storage if I had more disk/space. I can also control services for as far as I've explored.

The buttons for these items are greyed out, and when hovering over them it says "User John Doe is not permitted to..."

**Comment by stefwalter** [comment]
This is indeed an issue. Cockpit does indeed hard code the list of groups that it uses to enable parts of the UI:

https://github.com/cockpit-project/cockpit/blob/master/pkg/base1/cockpit.js#L3717

That's a bug. We need to ask polkit about this instead.

**Comment by AceBlade258** [comment]
IDK how particularly involved that is, but based on the way that is written now, would it be easier to add an option to choose which groups Cockpit recognizes as admin?

Unless I'm mistaken, changing

`if (admin && (name == "wheel" || name == "sudo")) {`

to

`if (admin && (name == "wheel" || name == "sudo" || name == "admins")) {`

Would handle it in the case of FreeIPA.

**Comment by Gundersanne** [comment]
I think this got fixed in #9127, feel free to reopen if that's not the case.
